db/seeds.rb

```diff
@@ -40,22 +74,30 @@ def put_user_credentials(username, password)
   email: 'musician@example.com',
   password: musician_password,
   password_confirmation: musician_password,
-  greenfield_enabled: true
+  greenfield_enabled: true,
+  current_login_ip: 'fc00:1:1::3'
```

Comment:
Hehe, what's this ipv6 addy?

Reply:
It's the equivalent of `10.0.1.3`, see: https://en.wikipedia.org/wiki/Unique_local_address